images/router/haproxy/conf/haproxy-config.template

```diff
@@ -6,7 +6,8 @@
 {{ define "/var/lib/haproxy/conf/haproxy.config" }}
 {{ $workingDir := .WorkingDir }}
 global
-  # maxconn 4096
+  maxconn {{env "ROUTER_MAX_CONNECTIONS" "20000"}}
```

Comment:
@smarterclayton a maxconns=auto type thing would be an RFE against haproxy.  The idea makes sense to me, I wonder if it's been discussed upstream before.